pkgs/servers/web-apps/vikunja/frontend.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, unzip, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "vikunja-frontend";
+  version = "0.17.0";
+  src = fetchurl {
+    url = "https://dl.vikunja.io/frontend/${pname}-${version}.zip";
```

Comment:
Can we compile this ourselves?

Reply:
It is certainly possible with yarn2nix ([example](https://github.com/jtojnar/nixfiles/blob/28a348c486e749f15be23f59e7cbea8efb08378f/pkgs/vikunja/vikunja-frontend/default.nix)). But there is a bug: https://github.com/nix-community/yarn2nix/issues/173

Reply:
I pushed a version based on @jtojnar's example. Maybe you can have a look at it and check if it's good.

Reply:
My attempt at packaging is [here](https://kolaente.dev/vikunja/desktop-nix/src/branch/main/frontend/default.nix) but it has the wired error from the bug @jtojnar mentioned. I did have some success building it on 21.05 but not reliably.

Reply:
Do you think it would be better to use the prebuild version at the moment and switch to a self-build version later when it's more stable (I don't know how unstable it is because I didn't experience the issue myself)?

Reply:
If it worked for you then we should be fine building this ourselves.

Reply:
I have just taken some time to test again exactly and then this error also occurred several times. Therefore I would suggest to take the pre-build variant again, if not someone has an idea for a fix.

Reply:
I decided to use the pre-build version until someone has a fix